README.rst

```diff
@@ -216,31 +218,46 @@ IP addresses usage:
 .. code:: yaml
 
   zookeeper:
-    nodes:
-      - 192.168.0.101
-      - 192.168.0.102
-      - 192.168.0.103
-
+    clusters:
+      - nodes:
+        - 192.168.0.101
+        - 192.168.0.102
+        - 192.168.0.103
+      - nodes:
+        - 192.168.1.101
+        - 192.168.1.102
+        - 192.168.1.103
+        
 Minion id usage:
 
 .. code:: yaml
 
   zookeeper:
-    nodes:
-      - minion1
-      - minion2
-      - minion3
+    clusters:
+      - nodes:
+        - minion1
+        - minion2
+        - minion3
+      - nodes:
+        - minion4
+        - minion5
+        - minion6
 
 Mixed usage (IP, minion id, fqdn):
 
 .. code:: yaml
 
   zookeeper:
-    nodes:
-      - 192.168.0.101
-      - minion2
-      - zookeeper3.mysite.com
-      
+    clusters:
+      - nodes:
+        - 192.168.0.101
+        - minion2
+        - zookeeper3.cluster1.mysite.com
+      - nodes:
+        - 192.168.1.101
+        - minion-hostname5
+        - zookeeper3.cluster2.mysite.com
```

Comment:
I wonder, what would happen if you specify the same Minion (perhaps using different targeting method) in multiple clusters?